SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -66,6 +68,7 @@ ASpatialDebugger::ASpatialDebugger(const FObjectInitializer& ObjectInitializer)
 	{
 		NetDriver->SetSpatialDebugger(this);
 	}
+	HoverIndex = 0;
```

Comment:
uber nit: set this up with the other variables

Reply:
Still inside the constructor but further up?